packages/strapi-design-system/src/Accordion/AccordionToggle.js

```diff
@@ -109,19 +110,19 @@ export const AccordionToggle = ({ title, description, as, togglePosition, action
           >
             <>
               {size === 'S' ? (
-                <Text bold={true} as={as} id={ariaLabelId} textColor={titleColor}>
+                <Typography fontWeight="bold" as={as} id={ariaLabelId} textColor={titleColor}>
                   {title}
-                </Text>
+                </Typography>
               ) : (
-                <H3 as={as} id={ariaLabelId} textColor={titleColor}>
+                <AccordionTypography variant="delta" as={as} id={ariaLabelId} textColor={titleColor}>
```

Comment:
Just a thought about the naming but wdyt about `DeltaTypography` as name for this component?

Reply:
This is a specific case to help the accordion to select the right css class. I name it AccordionTypography because it shouldn't be used outside the accordion. If we rename it `<DelaTypography variant="delta" />`, the name wouldn't fit if we change the variant eg `<DeltaTypography variant="beta" />` 🤔